dags/atd_knack_traffic_detectors_weekly_snapshot.py

```diff
@@ -0,0 +1,120 @@
+# test locally with: docker compose run --rm airflow-cli dags test atd_knack_traffic_detectors_weekly_snapshot
```

Comment:
thanks for these helpful testing breadcrumbs, and doubly so for doing them with the compose stack -- that's 💯